slf4j/src/main/scala/org/typelevel/log4cats/slf4j/internal/Slf4jLoggerInternal.scala

```diff
@@ -37,10 +37,11 @@ private[slf4j] object Slf4jLoggerInternal {
   private[this] def contextLog[F[_]](
       isEnabled: F[Boolean],
       ctx: Map[String, String],
-      logging: () => Unit
+      logging: () => Unit,
+      sync: Sync.Type
   )(implicit F: Sync[F]): F[Unit] = {
 
-    val ifEnabled = F.delay {
+    val ifEnabled = F.suspend(sync) {
```

Comment:
Sorry, I think this is another one that should stay as `delay`. Interacting with the `MDC` should not be blocking, it's just a thread-local mutable map.

Reply:
Done. Let me know if there is something else.